v2/pkg/reporting/trackers/github/github.go

```diff
@@ -88,12 +90,64 @@ func (i *Integration) CreateIssue(event *output.ResultEvent) error {
 		labels = append(labels, label)
 	}
 
-	req := &github.IssueRequest{
-		Title:     &summary,
-		Body:      &description,
-		Labels:    &labels,
-		Assignees: &[]string{i.options.Username},
+	ctx := context.Background()
+
+	var existingIssue *github.Issue
+	if i.options.ReuseIssues {
+		existingIssue, err = i.findIssueByTitle(ctx, summary)
+		if err != nil && !errors.Is(err, io.EOF) {
+			return err
+		}
+	}
+
+	if existingIssue == nil {
+		req := &github.IssueRequest{
+			Title:     &summary,
+			Body:      &description,
+			Labels:    &labels,
+			Assignees: &[]string{i.options.Username},
+		}
+		_, _, err = i.client.Issues.Create(ctx, i.options.Owner, i.options.ProjectName, req)
+		return err
+	} else {
+		req := &github.IssueComment{
+			Body: &description,
+		}
+		_, _, err = i.client.Issues.CreateComment(ctx, i.options.Owner, i.options.ProjectName, *existingIssue.Number, req)
+		return err
+	}
+}
+
+func (i *Integration) findIssueByTitle(ctx context.Context, title string) (*github.Issue, error) {
+	req := &github.IssueListByRepoOptions{
+		State:     "open",
```

Comment:
How about re-opening the closed ones? See: https://github.com/projectdiscovery/nuclei/pull/4152/files#diff-395c2e5b2b284021a59744f7d26fc07bcd1b2ed0178fa9cedc6e1467f4dba09fR95